dev-tools/omdb/src/bin/omdb/db.rs

```diff
@@ -2274,9 +2259,23 @@ async fn cmd_db_disk_info(
     println!("{}", table);
 
     get_and_display_vcr(disk.volume_id(), datastore).await?;
+
     Ok(())
 }
 
+/// Run `omdb db disk info <UUID>`.
+async fn cmd_db_disk_info(
+    opctx: &OpContext,
+    datastore: &DataStore,
+    args: &DiskInfoArgs,
+) -> Result<(), anyhow::Error> {
+    match datastore.disk_get(opctx, args.uuid).await? {
+        Disk::Crucible(disk) => {
+            crucible_disk_info(opctx, datastore, disk).await
+        }
```

Comment:
Ah, this is where we will update things to display the new disks types.  cool cool.